src/lib/import_export/binary/binary_parser.cpp

```diff
@@ -240,6 +247,48 @@ std::shared_ptr<FrameOfReferenceSegment<T>> BinaryParser::_import_frame_of_refer
   return std::make_shared<FrameOfReferenceSegment<T>>(block_minima, null_values, std::move(offset_values));
 }
 
+std::shared_ptr<FSSTSegment<pmr_string>> BinaryParser::_import_fsst_segment(std::ifstream& file,
+                                                                            ChunkOffset row_count) {
+  const auto compressed_vector_type_id = _read_value<CompressedVectorTypeID>(file);
+
+  const auto compressed_value_size = _read_value<uint32_t>(file);
+  pmr_vector<unsigned char> compressed_values(_read_values<unsigned char>(file, compressed_value_size));
+
+  auto offset_values = _import_offset_value_vector(file, row_count + 1, compressed_vector_type_id);
```

Comment:
const?

Reply:
Can't be const, as we're moving the value in the FSST constructor.